tests/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Configuration.Tests/ConfigUpdaterStepTests.cs

```diff
@@ -0,0 +1,215 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Autofac;
+using Autofac.Extras.Moq;
+using Microsoft.Extensions.Logging.Abstractions;
+using Moq;
+using Xunit;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Steps.Configuration.Tests
+{
+    public class ConfigUpdaterStepTests
+    {
+        [Fact]
+        public void CtorTests()
+        {
+            // Arrange
+            using var mock = AutoMock.GetLoose(cfg =>
+            {
+                cfg.RegisterInstance(new ConfigUpdaterOptions
+                {
+                    ConfigFilePaths = new[] { "a", "b" }
+                });
+                RegisterConfigUpdaters(cfg, 0, 2);
+            });
+            var step = mock.Create<ConfigUpdaterStep>();
+
+            // Act
+            var dependencyOf = step.DependencyOf;
+            var dependsOn = step.DependsOn;
+            var description = step.Description;
+            var id = step.Id;
+            var title = step.Title;
+            var subSteps = step.SubSteps;
+            var status = step.Status;
+            var done = step.IsDone;
+
+            // Assert
+            Assert.Equal(new[] { "Microsoft.DotNet.UpgradeAssistant.Steps.Solution.NextProjectStep" }, dependencyOf);
```

Comment:
Should we keep a list of "well known steps" that we can just reference via a static class? These magic strings get difficult

Reply:
That's a good idea. I'll add this.